identity/README.md

```diff
@@ -0,0 +1,49 @@
+# identity example
+Example using chrome.identity with facebook and google.
+
+The use of the identity api can be found in the authorize function in
+background.js.  The rest of the code provides a working example of using
+oauth with Facebook and Google.
+
+# First time run before setting up applications.
+
+You will want to set a specific ID in your manifest.json file.  In this example,
+it is set to identity.example@mozilla.org.  The identity API has a helper
+function that generates a hashed URL based on the ID.  If you do not have a
+specific ID set, the extension id will change every time you install the
+extension (or reload as a temporary addon).
+
+Load the extension as a temporary addon in about:debugging then open the
+Brower console and look for a message: Your redirect_uri is XXX.  This message
```

Comment:
Ugh, I found this hard to find amongst the noise. Would it be possible to place on `index.html`? And we could put a few more of these docs there too? Basically just means adding something like:

`document.getElementById('redirect_url').innerText = browser.identity.getRedirectURL("callback");`

..to index.js